git_pw/config.py

```diff
@@ -25,6 +26,9 @@ def _get_config(key):
     except subprocess.CalledProcessError:
         output = ''
 
+    if os.environ.get('PW_%s' % key.upper()) is not None:
```

Comment:
Kind of. I was figuring we could treat `git_pw.config.Config` as a singleton and set the `username` and `password` attributes if they were present. The `__getattr__` function would then only load from the config file if not already set. Would that work?